### ISSUE TITLE
No logs for pre/post execution scripts

### ISSUE THREAD

**Issue by CJHoffmann** [Luxoft/Twister#6]
There is no indication in any of the logs if a pre or post execution script has been executed.
In addition, any CLI interaction or print statements in the script file are not displayed anywhere.
Since this sort of script is used for system initialization, it is necessary to log the start and end of the script and any CLI interactions.

**Comment by croqaz** [comment]
Hello, this issue was fixed in Release_2.002 - Update on 17th May.
You can see the CLI interaction in the Central Engine REST Log, in your browser, using the ip + port of the CE and adding "/rest" at the end (ex address: http://127.0.0.1:8000/rest/#tab_logs).

**Comment by bogdanpopescu** [comment]
The indication about pre and post execution scripts is in the CE log.
@CJHoffmann please verify it with 31_05_2013 tag.

**Comment by sandhyamisra** [comment]
The logs for the pre and post execution scripts are dumped as a part of CE execution log [ /opt/twister/ce_log.log ] . It is difficult to debug.

 Expectation : We need separate execution log for pre and post test scripts in the client log directory. So that it will be easy for us to figure out for which EP/user the pre and post execution scripts runs[ During parallel execution]

**Comment by sandhyamisra** [comment]
This issue is still not resolved. 

The logs are not getting created in the EP_100*.log they are still getting created at the server. 

/opt/twister/server_logs.log
/opt/twister/logs/*

**Comment by CJHoffmann** [comment]
Tested with GUI V2.021

I sometimes see in my $HOME/twister directory a log_srv.log file that looks like the CE log for a test run.
I deleted this file, expecting that a new one would be recreated for the next test run.  It did not.
I created an empty log_srv.log file, but it remained empty after a test run.

When or how is this log file created?
It seems it would satisfy this issue, if the log would be created for each test run and if it could be put in the logs directory.
